lib/uplink/bucket.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplink
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// Bucket represents operations you can perform on a bucket
+type Bucket struct {
```

Comment:
Shouldn't we have a field for bucket name here? Or maybe I don't understand the purpose of this `Bucket` struct.

Reply:
we should have storj.Bucket embedded probably, yeah, i'll fix